src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.vb

```diff
@@ -73,7 +74,7 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                 If m_ControlData Is Nothing Then
 
                     TargetFrameworkPropertyControlData = New TargetFrameworkPropertyControlData(
```

Comment:
This will break .NET Framework and other legacy projects. We need to branch this and go down the old path, probably based on whether SupportedTargetFrameworks property is present.